src/main.tsx

```diff
@@ -0,0 +1,10 @@
+import { StrictMode } from 'react'
```

Comment:
**praise**: Great use of React 18's createRoot API for rendering the application.